crate/guides/quickstart.md

```diff
@@ -26,21 +26,25 @@ or `todomvc` examples.
 
 ## Setup
 
-This framework requires you to install [Rust](https://www.rust-lang.org/tools/install).
+The two fundamental things to setup:
 
-You'll need a recent version of Rust: `rustup update`
+ * [Rust](https://www.rust-lang.org/tools/install), the language itself.
+  * `rustup update` is also important
+ * `cargo install cargo-make`, a broadly useful tool to for Rust [workflows](https://github.com/sagiegurari/cargo-make)
```

Comment:
There is missing dot at the end even though the sentence doesn't continue on the next line (like the case below).
We should make content as consistent as possible to make it more readable / scannable.